en-US/firefox/family/index.lang

```diff
@@ -1,23 +1,33 @@
 ## ios_product_tagline ##
 ## tracking_protection ##
-## fx49_remove_hello ##
 ## download_browse_freely ##
+## firefox_title_meta_bug1345338 ##
 ## NOTE: demo page available at https://www-dev.allizom.org/firefox/products/
 ## URL: https://www-dev.allizom.org/%LOCALE%/firefox/products/
 
 
-## TAG: fx49_remove_hello
+## TAG: firefox_title_meta_bug1345338
+# HTML page title
+;Free Web Browser for Android, iOS, Desktop | Firefox
+Free Web Browser for Android, iOS, Desktop | Firefox
+
+
 # Title of the page (HTML meta tag)
```

Comment:
Add obsolete note, also to the description (` Learn more about our complete lineup of products for desktop and mobile browsing (and more) that help keep you in control online. Free yourself with Firefox.`)